### PR TITLE
Group popup menu items

### PR DESCRIPTION
**Pull request by barmac**
Required for https://github.com/bpmn-io/bpmn-js/pull/1682

### CODE REVIEW

File: lib/features/popup-menu/PopupMenu.js [hunk 1]
```diff
@@ -501,8 +504,16 @@ PopupMenu.prototype._createEntries = function(entries, className) {
...
 
   forEach(entries, function(entry, id) {
-    var entryContainer = self._createEntry(entry, id);
-    entriesContainer.appendChild(entryContainer);
+    var entryContainer = self._createEntry(entry, id),
+        grouping = entry.group || 'default',
+        groupContainer = domQuery('[data-group=' + grouping + ']', entriesContainer);
+
+    if (!groupContainer) {
+      groupContainer = domify('<div class="group" data-group="' + grouping + '"></div>');
```
Comment (nikku) [inline lib/features/popup-menu/PopupMenu.js:512]: We should escape `grouping` to not allow XSS.

**Review by nikku — CHANGES_REQUESTED** [review]

File: lib/features/popup-menu/PopupMenu.js (hunk 1 above)
Comment (barmac) [reply lib/features/popup-menu/PopupMenu.js:512]: Valid point. We will need to fix it in [the context pad](https://github.com/bpmn-io/diagram-js/blob/develop/lib/features/context-pad/ContextPad.js#L286) as well.

Comment (barmac) [reply lib/features/popup-menu/PopupMenu.js]: This is now fixed, but there are also a couple of other places where we allow DOM injection. I will fix them.

Comment (nikku) [reply lib/features/popup-menu/PopupMenu.js]: Woops woops :).

**Pull request merged**